js/index.js

```diff
@@ -0,0 +1,37 @@
+
+  // var h = $("header").height();
+  // $(".main__nav").height(h);
+  // $(".socialmedia").height(h);
```

Comment:
Using comments for pseudo code or notes is a good thing, but leaving old code may eventually become confusing. Unless you're saving this for later I'd recommend removing it.